assets/js/sticky-menu.js

```diff
@@ -58,7 +59,7 @@ define( ['jquery', 'underscore'], function ( $, _ ) {
 
 			$( window ).on( 'scroll.ptStickyMenu', _.bind( _.throttle( function() {
 				// check for new state
-				newMenuState = this.isScrollDirectionUp() && this.isWindowTopBellowOffset();
+				newMenuState = ( this.isScrollDirectionUp() || this.isAllDirectionEnabled() ) && this.isWindowTopBellowOffset() ;
```

Comment:
@krumpak There is a single space at the end of this line, just before the semicolon, which should be removed.